server/src/services/auth.service.ts

```diff
@@ -185,6 +185,7 @@ export class AuthService {
   }
 
   async signup(email: string) {
+    email = email ? email.toLowerCase() : '';
```

Comment:
```suggestion
    email = email?.toLowerCase();
```

Reply:
please check if we can do it in dto transformations

Reply:
We can't do this through transformation for AppAuthenticationDto. cause that same dto is using with login function too. so that will cause `user not found or unauthorized` errors if an existed user with email that has Uppercase letters

Reply:
I can create a new different dto for create controller function